fast_api/models.py

```diff
@@ -501,3 +501,8 @@ class EvaluationRunDeletionBody(BaseModel):
 class SearchQuestionReformulationBody(BaseModel):
     question: StrictStr
     apiKey: StrictStr
+
+
+class InviteUsersBody(BaseModel):
+    emails: List[StrictStr]
+    organization_name: StrictStr
```

Comment:
maybe organization_id? Should reduce the requests we need to do for it to work :)

Reply:
we use 'update_organization_of_user' with a organization_name as a parameter to update the organization of a user (dropdown on users page), that's why i used the name of the org, we can of course change it?

Reply:
works for me, either way is fine, in theory a name based selection will take longer but since we have max 100 orgs it's a micro optimization so can be skipped :)